src/components/Section/Print/Print.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import MockAdapter from 'axios-mock-adapter'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+import { Provider } from 'react-redux'
+import { mount } from 'enzyme'
+import Print from './Print'
+
+const applicationState = {
+  Application: {}
+}
+
+describe('The print section', () => {
+  // Setup
+  window.token = 'fake-token'
+  const middlewares = [ thunk ]
+  const mockStore = configureMockStore(middlewares)
+
+  it('visible when authenticated', () => {
+    const store = mockStore({ authentication: { authenticated: true, twofactor: true, application: applicationState } })
+    const component = mount(<Provider store={store}><Print /></Provider>)
+    expect(component.find('.section-print-container').length).toBe(10)
+  })
```

Comment:
Is it possible to mock `window.print` here, simulate a print button click, and verify that `window.print()` was called?  I'm not sure if that's something you can do with enzyme but if so, that'd be a nice addition.

Reply:
We should be able to and great idea! Not sure if it'll be directly using enzyme but we've done similar things in the past which we can leverage.